.github/workflows/ci.yml

```diff
@@ -191,26 +191,58 @@ jobs:
 
   build-macos:
     name: macOS Build
-    runs-on: macOS-11.0
+    runs-on: macOS-11
+    env:
+      QT_DEST: Qt
     steps:
       - uses: actions/checkout@v2
       - name: Select Latest Xcode
         run: |
           set -xe
           sw_vers
           ls -d /Applications/Xcode*.app
-          sudo xcode-select --switch /Applications/Xcode_11.7.app
+          LAST_XCODE=$(ls -d /Applications/Xcode*.app | grep -v beta | sort -Vr | head -n1)
+          sudo xcode-select --switch $LAST_XCODE
           xcode-select -p
           xcodebuild -version
           xcodebuild -showsdks
-      - run: brew update && brew install qt5
+      - name: install Qt installer
+        run: |
+          set -xe
+          pip3 install -U pip
+          pip3 install aqtinstall
+          pip3 show aqtinstall
+          aqt list-qt mac desktop
+          QT_VERSION=$(aqt list-qt mac desktop --spec 6 --latest-version)
+          echo "QT_VERSION=$QT_VERSION" >> $GITHUB_ENV
+      - name: Cached Qt
+        id: cache-qt
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.QT_DEST }}
```

Comment:
In https://github.com/psieg/Lightpack/pull/498/checks?check_run_id=3862943206, the cache is not hit and uploads an entry on completion, however the entry is empty (22B). Unfortunately it doesn't say which path it used. 

I'm not that familiar with GH actions, but none of the examples uses env variables in path (only in key). Did you try putting Qt here?

Reply:
I tested on another branch. This path is not relative to the working directory configured on top. I'll fix after merging. Thanks!